.github/workflows/tests.yaml

```diff
@@ -36,6 +36,10 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y bats
 
+          # Install bats
+          sudo apt-get update
+          sudo apt-get install -y bats
```

Comment:
These lines are duplicated.

Reply:
Hey @rst0git I need to rebase this pr, don't look at this right now, I am gonna open it and inform you for the review.